src/main/java/lilylicious/staticchunkmanager/chunk/ChunkRequest.java

```diff
@@ -46,6 +47,15 @@ public ChunkRequest(InputStream is, int x, int y, boolean... extras) {
         this.matchBiomes = extras.length > 0 ? extras[0] : false;
     }
 
+    public ChunkRequest(InputStream is, String path, int x, int y, boolean... extras) {
```

Comment:
Adjust this in accordance with https://github.com/Lilylicious/StaticChunkManager/blob/master/src/main/java/lilylicious/staticchunkmanager/chunk/ChunkRequest.java#L39

Other than that, it looks fine